rabbitmq/metadata.csv

```diff
@@ -78,223 +78,226 @@ rabbitmq.overview.messages.publish_in.count,gauge,,message,,Count of messages pu
-rabbitmq.io.write_bytes.count,count
-rabbitmq.io.write_ops.count,count
-rabbitmq.io.write_time_seconds.count,count
-rabbitmq.msg_store.read.count,count
-rabbitmq.msg_store.write.count,count
-rabbitmq.process.max_fds,gauge
-rabbitmq.process.max_tcp_sockets,gauge
-rabbitmq.process.open_fds,gauge
-rabbitmq.process.open_tcp_sockets,gauge
-rabbitmq.process.resident_memory_bytes,gauge
-rabbitmq.process_start_time_seconds,gauge
-rabbitmq.queue.consumer_capacity,gauge
-rabbitmq.queue.disk_reads.count,count
-rabbitmq.queue.disk_writes.count,count
-rabbitmq.queue.index.read_ops.count,count
-rabbitmq.queue.index.write_ops.count,count
-rabbitmq.queue.messages.bytes,gauge
-rabbitmq.queue.messages.paged_out_bytes,gauge
-rabbitmq.queue.messages.paged_out,gauge
-rabbitmq.queue.messages.persistent,gauge
-rabbitmq.queue.messages.persistent_bytes,gauge
-rabbitmq.queue.messages.published.count,count
-rabbitmq.queue.messages.ram_bytes,gauge
-rabbitmq.queue.messages.ram,gauge
-rabbitmq.queue.messages.ready_bytes,gauge
-rabbitmq.queue.messages.ready_ram,gauge
-rabbitmq.queue.messages.ready,gauge
-rabbitmq.queue.messages.unacked_bytes,gauge
-rabbitmq.queue.messages.unacked_ram,gauge
-rabbitmq.queue.messages.unacked,gauge
-rabbitmq.queue.process_memory_bytes,gauge
-rabbitmq.queue.process_reductions.count,count
-rabbitmq.queues.created.count,count
-rabbitmq.queues.declared.count,count
-rabbitmq.queues.deleted.count,count
-rabbitmq.queues,gauge
-rabbitmq.raft.entry_commit_latency_seconds,gauge
-rabbitmq.raft.log.commit_index,gauge
-rabbitmq.raft.log.last_applied_index,gauge
-rabbitmq.raft.log.last_written_index,gauge
-rabbitmq.raft.log.snapshot_index,gauge
-rabbitmq.raft.term.count,count
-rabbitmq.resident_memory_limit_bytes,gauge
-rabbitmq.schema.db.disk_tx.count,count
-rabbitmq.schema.db.ram_tx.count,count
-rabbitmq.telemetry.scrape.duration_seconds.count,count
-rabbitmq.telemetry.scrape.duration_seconds.sum,count
-rabbitmq.telemetry.scrape.encoded_size_bytes.count,count
-rabbitmq.telemetry.scrape.encoded_size_bytes.sum,count
-rabbitmq.telemetry.scrape.size_bytes.count,count
-rabbitmq.telemetry.scrape.size_bytes.sum,count
+rabbitmq.alarms.file_descriptor_limit,gauge,,,,[OpenMetricsV2] is 1 if file descriptor limit alarm is in effect,0,rabbitmq,file descriptor limit,
+rabbitmq.alarms.free_disk_space.watermark,gauge,,,,[OpenMetricsV2] is 1 if free disk space watermark alarm is in effect,0,rabbitmq,free disk space watermark,
+rabbitmq.alarms.memory_used_watermark,gauge,,,,[OpenMetricsV2] is 1 if VM memory watermark alarm is in effect,0,rabbitmq,memory used watermark,
+rabbitmq.auth_attempts.count,count,,,,[OpenMetricsV2] Total number of authorization attempts,0,rabbitmq,auth attempts,
+rabbitmq.auth_attempts.failed.count,count,,,,[OpenMetricsV2] Total number of failed authentication attempts,0,rabbitmq,auth attempts failed,
+rabbitmq.auth_attempts.succeeded.count,count,,,,[OpenMetricsV2] Total number of successful authentication attempts,0,rabbitmq,auth attempts succeeded,
```

Comment:
Weird that the descriptions aren't consistent with `authorization` vs `authentication`, but I guess that is what the rabbitmq metric descriptions are: 

```
# TYPE rabbitmq_auth_attempts_total counter
# HELP rabbitmq_auth_attempts_total Total number of authorization attempts
rabbitmq_auth_attempts_total{protocol="http"} 40
# TYPE rabbitmq_auth_attempts_succeeded_total counter
# HELP rabbitmq_auth_attempts_succeeded_total Total number of successful authentication attempts
rabbitmq_auth_attempts_succeeded_total{protocol="http"} 40
# TYPE rabbitmq_auth_attempts_failed_total counter
# HELP rabbitmq_auth_attempts_failed_total Total number of failed authentication attempts
rabbitmq_auth_attempts_failed_total{protocol="http"} 0
```

Reply:
Nice catch! I've made a note of this and will bring it up with rabbitmq folks. If that's something they decide to fix, we can update it here, for now I'd stick with what they say in case there is a difference.